src/AstroImages.jl

```diff
@@ -4,7 +4,7 @@ module AstroImages
 
 using FITSIO, FileIO, Images, Interact, Reproject, WCS, MappedArrays
 
-export load, AstroImage, ccd2rgb
+export load, AstroImage, ccd2rgb, set_brightness!, set_contrast!, add_label!, reset!
```

Comment:
Do you expect users to directly call these functions?  If not, why exporting them?

Reply:
Yes, they will be called directly by the user.